dstep/main.d

```diff
@@ -59,8 +59,9 @@ auto parseCLI (string[] args)
         std.getopt.config.passThrough,
         std.getopt.config.caseSensitive,
         "output|o", &config.output,
-        "language|x", &parseLanguage,
         "objective-c", &forceObjectiveC,
+        "language|x", "Treat subsequent input files as having type <language>.",
```

Comment:
Actually, you could add the help snippet to `customEntries` at `main.cpp:226` instead.